javascript/brahma-firelight/package.json

```
@@ -1,6 +1,6 @@
 {
   "type": "module",
   "dependencies": {
-    "brahma-firelight": "^1.5.2"
+    "brahma-firelight": "1.5.2"
```

```suggestion
    "brahma-firelight": "^1.5.2"
```

better to avoind sticky version 😛